justfile

```diff
@@ -69,9 +69,9 @@ alias l := lint
 
 [no-exit-message]
 _lint-go:
-    cd modules/cli && golangci-lint run
-    cd modules/go-sequencer-client && golangci-lint run
-    cd modules/bech32m && golangci-lint run
+    cd modules/cli && golangci-lint run ./...
```

Comment:
This change is equivalent to what was already there
https://golangci-lint.run/welcome/quick-start/